src/NServiceBus.Core.Tests/ApprovalFiles/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -1,5 +1,6 @@
-[assembly: System.CLSCompliantAttribute(true)]
+[assembly: System.CLSCompliantAttribute(false)]
```

Comment:
Maybe worth updating https://github.com/Particular/PlatformArchitecture/blob/7aec17ec4abde9069e9e9a4f9ea3a346a021d2ff/repo-standards/README.md#cls-compliance ?

Reply:
yeah will be done once merged